SQL/New_patches/2020-04-01-GUID.sql

```diff
@@ -0,0 +1,16 @@
+ALTER Table Project ADD COLUMN GUIDRequired binary NOT NULL DEFAULT 0;
```

Comment:
I think you need to add these changes to RB. Travis is choking on it.

Reply:
@john asmentioned above this PR was not meant to be merged but instead was meant as a proposal.

several things have to be done to make it meargeable including putting it on the correct branch

Reply:
OK cool. In general I am in favour of adding this, but it should be done as a dependency as I mentioned in another comment.

The "token" stuff should probably be removed because I don't think it's done properly here. We can figure out how to connect to external servers in another way, probably using Guzzle or the API instead of the way it's implemented now.